frontend/src/component/playground/Playground/PlaygroundEnvironmentTable/PlaygroundEnvironmentTable.tsx

```diff
@@ -0,0 +1,134 @@
+import { useMemo } from 'react';
+import {
+    useFlexLayout,
+    useGlobalFilter,
+    useSortBy,
+    useTable,
+} from 'react-table';
+
+import { VirtualizedTable } from 'component/common/Table';
+import { sortTypes } from 'utils/sortTypes';
+import {
+    AdvancedPlaygroundEnvironmentFeatureSchema,
+    PlaygroundFeatureSchema,
+} from 'openapi';
+import { useMediaQuery, useTheme } from '@mui/material';
+import { useConditionallyHiddenColumns } from 'hooks/useConditionallyHiddenColumns';
+import { FeatureStatusCell } from '../PlaygroundResultsTable/FeatureStatusCell/FeatureStatusCell';
+import { FeatureResultInfoPopoverCell } from '../PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureResultInfoPopoverCell';
+import { VariantCell } from '../PlaygroundResultsTable/VariantCell/VariantCell';
+import { HighlightCell } from '../../../common/Table/cells/HighlightCell/HighlightCell';
+
+interface IPlaygroundEnvironmentTableProps {
+    features: AdvancedPlaygroundEnvironmentFeatureSchema[];
+}
+
+export const PlaygroundEnvironmentTable = ({
+    features,
+}: IPlaygroundEnvironmentTableProps) => {
+    const theme = useTheme();
+    const isExtraSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
+    const dynamicHeaders = Object.keys(features[0].context)
+        .filter(contextField => contextField !== 'appName')
+        .map(contextField => ({
+            Header: contextField,
```

Comment:
I would capitalize the first letter of the context field for the Header

Reply:
I will check with Nico because in the design he had lowercase and I'm not sure whether it's not gonna be confusing since the context uses lower cased ones.